### PR TITLE
Add a link to GLAM for the selected metric

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: src/pages/MetricDetail.svelte [hunk 1]
```diff
@@ -10,6 +10,34 @@
...
+    };
+    if (Object.keys(map).includes(app)) {
+      const p = map[app];
+      return function makeGlamUrl(metric) {
+        return `https://glam.telemetry.mozilla.org/${p.product}/probe/${metric}/explore?app_id=${p.app_id}`;
+      };
+    }
+
+    // The app isn't one GLAM supports so return nothing.
+    return null;
```
Comment (wlach) [inline src/pages/MetricDetail.svelte:36]: `undefined` is a little more idiomatic here

```suggestion
    return undefined;
```

File: src/pages/MetricDetail.svelte [hunk 2]
```diff
@@ -197,6 +225,15 @@
         {/each}
       </td>
     </tr>
+    {#if glamUrl}
+      <tr>
+        <td>GLAM</td>
+        <td>
+          <a
+            href={glamUrl(metric.bigquery_names.glam_etl_name)}>{glamUrl(metric.bigquery_names.glam_etl_name)}</a>
```
Comment (wlach) [inline src/pages/MetricDetail.svelte:233]: Showing the full URL in the UI seems a little verbose (you can probably skip the `const` declaration above if you take it out, as well).

```suggestion
            href={glamUrl(metric.bigquery_names.glam_etl_name)}>{metric.bigquery_names.glam_etl_name}</a>
```

**Review by wlach — CHANGES_REQUESTED** [review]
This looks and works great! Just a few minor changes and we'll be good to land.

File: src/pages/MetricDetail.svelte [hunk 3]
```diff
@@ -197,6 +225,15 @@
         {/each}
       </td>
     </tr>
+    {#if glamUrl}
+      <tr>
+        <td>GLAM</td>
```
Comment (robhudson) [inline src/pages/MetricDetail.svelte:230]: Would you prefer this titled "GLAM URL" instead?

Comment (wlach) [reply src/pages/MetricDetail.svelte:230]: Nope! I think just `GLAM` is fine. We can add a help icon when @Fenn-CS finishes #293

File: src/pages/MetricDetail.svelte (hunk 2 above)
Comment (robhudson) [reply src/pages/MetricDetail.svelte]: I updated this but still need the `const` for the `{#if}` check to show the table row.

**Review by wlach — APPROVED** [review]
Looks great, thanks!

File: src/pages/MetricDetail.svelte [hunk 4]
```diff
@@ -197,6 +225,15 @@
         {/each}
       </td>
     </tr>
+    {#if glamUrl}
```
Comment (wlach) [inline src/pages/MetricDetail.svelte:228]: Argh, sorry, did some last minute testing and it looks like this doesn't do the right thing for events (it incorrectly suggests we can view them in GLAM). Maybe the suggestion below will fix that?

```suggestion
    {#if glamUrl && metric.type === "event"}
```

**Review by wlach — CHANGES_REQUESTED** [review]

File: src/pages/MetricDetail.svelte (hunk 4 above)
Comment (robhudson) [reply src/pages/MetricDetail.svelte]: Cool, thanks. I added a condition similar to one just above it. Let me know if that's correct. It worked locally for me.

**Review by wlach — APPROVED** [review]
Awesome. I'm going to hold off on landing this until #312 is merged so as not to cause unnecessary merge conflicts.

**Pull request merged**